tests/integration/benchmark_runner/common/oc/test_oc.py

```diff
@@ -107,7 +107,7 @@ def test_oc_get_free_disk_id():
     oc = OC(kubeadmin_password=test_environment_variable['kubeadmin_password'])
     oc.login()
     with mock.patch.object(OC, 'get_worker_disk_ids', new=mock_get_worker_disk_ids):
-        assert oc.get_free_disk_id()
+        assert oc.get_free_disk_id(node='worker-2')
```

Comment:
Rather than having `worker-2` scattered around the tests, this belongs in the tests environment variables so that you can change this in just one place.

Reply:
Fixed